### DISCUSSION TITLE
Gists webhooks

### DISCUSSION THREAD

**Discussion by 9121999** [community/community#50394] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hello is there anyway or plan to create events/hooks for gists?
I would like to trigger pipelines when create or update a gist.

**Comment by maahaMaya** [comment, marked as answer]
you can use the API endpoints for listing, creating, updating, and deleting Gists and their comments. You can then use a webhook to receive notifications from GitHub when a Gist is created or updated, and trigger your pipeline accordingly.

Here is an example workflow:

Set up a webhook to receive notifications from GitHub when a Gist is created or updated.
When a webhook is triggered, use the GitHub API to retrieve information about the Gist.
Determine whether the Gist is a new Gist or an updated version of an existing Gist.
If the Gist is new, trigger your pipeline to process the Gist.
If the Gist is an update, trigger your pipeline to update the previously processed Gist.
Remember to follow best practices when using webhooks, such as verifying the signature of the webhook payload to ensure its authenticity.

Alternatively, you can use a third-party service such as Zapier or IFTTT to monitor your Gists for changes and trigger a pipeline when a new Gist is created or updated.

Keep in mind that there may be rate limits and other limitations on how frequently you can poll the GitHub API, so it's important to design your implementation accordingly.

**Reply by 9121999** [reply]
Thnak You

**Reply by cupnoodle** [reply]
How to set up a webhook for Gist created event? @maahaMaya
